lib/libmbedtls/core/ecc.c

```diff
@@ -349,20 +351,52 @@ static const struct crypto_ecc_keypair_ops ecc_keypair_ops = {
 	.shared_secret = &ecc_shared_secret,
 };
 
+static const struct crypto_ecc_keypair_ops sm2_pke_keypair_ops = {
+	.generate = &ecc_generate_keypair,
```

Comment:
We're normally not using a `&` when taking the address of a function.
I see that we're using it above, perhaps it's better to fix that there instead of spreading this?

Reply:
Agreed. I will fix the new code here, and open a separate PR to fix the existing occurrences.